src/element/collision.ts

```diff
@@ -786,7 +786,12 @@ export const findFocusPointForEllipse = (
       orientation * py * Math.sqrt(Math.max(0, squares - a ** 2 * b ** 2))) /
     squares;
 
-  const n = (-m * px - 1) / py;
+  let n = (-m * px - 1) / py;
+
+  if (n === 0) {
+    // if zero {-0, 0}, fall back to a same-sign value in the similar range
+    n = (Object.is(n, -0) ? -1 : 1) * 0.01;
+  }
```

Comment:
I've made it so that the `n` is in the similar vein of value we usually use here, and also so it's applied to computing `x` as well.

`Object.is(n, -0)` is needed because `Math.sign(n)` returns -0 for -0.